### PR TITLE
gh-92546: Move pprint benchmark into pyperformance

### PR DESCRIPTION
**Pull request by arhadthedev**
This PR couples with https://github.com/python/pyperformance/pull/222 and supersedes https://github.com/python/cpython/pull/92560. Inspired by https://github.com/python/cpython/issues/93096#issuecomment-1134576471.

* Issue: gh-92546

Automerge-Triggered-By: GH:ericsnowcurrently

### CODE REVIEW

File: Misc/NEWS.d/next/Library/2022-07-06-21-24-03.gh-issue-92546.s5Upkh.rst [hunk 1]
```diff
@@ -0,0 +1,2 @@
+An undocumented ``python -m pprint`` benchmark is moved into ``from_stdlib``
+suit of pyperformance. Patch by Oleg Iarygin.
```
Comment (hauntsaninja) [inline Misc/NEWS.d/next/Library/2022-07-06-21-24-03.gh-issue-92546.s5Upkh.rst]:
```suggestion
suite of pyperformance. Patch by Oleg Iarygin.
```

**Review by hauntsaninja — APPROVED** [review]
Had a nit, but looks good!

**Comment by arhadthedev** [comment]
Restart tests because `test.test_ssl.SimpleBackgroundTests.test_get_server_certificate_timeout` timeout failure seems unrelated.

**Comment by arhadthedev** [comment]
@ericsnowcurrently Could you look at this PR please? It removes the code fetched by https://github.com/python/pyperformance/pull/222.

File: Misc/NEWS.d/next/Library/2022-07-06-21-24-03.gh-issue-92546.s5Upkh.rst [hunk 2]
```diff
@@ -0,0 +1,2 @@
+An undocumented ``python -m pprint`` benchmark is moved into ``from_stdlib``
```
Comment (ericsnowcurrently) [inline Misc/NEWS.d/next/Library/2022-07-06-21-24-03.gh-issue-92546.s5Upkh.rst]:
```suggestion
An undocumented ``python -m pprint`` benchmark is moved into ``pprint``
```

**Review by ericsnowcurrently — APPROVED** [review]
LGTM, aside from one small change.

**Pull request merged**